### PR TITLE
Market recording mode

### PR DESCRIPTION
**Pull request by jorgegarcia7**
- Parameter to run flumine in Market Recording Mode, removing poll_account_balance and poll_market_closure calls since they do not add information to the recorded data.

### CODE REVIEW

File: flumine/flumine.py [hunk 1]
```diff
@@ -71,14 +63,23 @@ def _add_default_workers(self):
                 start_delay=10,  # wait for streams to populate
             )
         )
-        self.add_worker(
-            worker.BackgroundWorker(
-                self,
-                function=worker.poll_market_closure,
-                interval=60,
-                start_delay=10,  # wait for login
+        if self.client.market_recording_mode != True:
```
Comment (liampauling) [inline flumine/flumine.py:66]: Can you change to be more pythonic

```
if self.client.market_recording_mode:
```

**Review by liampauling — CHANGES_REQUESTED** [review]

File: flumine/flumine.py (hunk 1 above)
Comment (jorgegarcia7) [reply flumine/flumine.py:66]: This breaks the tests (test__add_default_workers), because flumine.client.market_recording_mode is an Object

Comment (liampauling) [reply flumine/flumine.py:66]: Sorry I mean

```python
if self.client.market_recording_mode is False:
```

And yeah the text will need to be fixed with

```python
self.flumine.client.market_recording_mode = False
```

Comment (jorgegarcia7) [reply flumine/flumine.py]: Great, thanks

**Review by liampauling — APPROVED** [review]

**Pull request merged**